include/nntile/kernel/embedding/cpu.hh

```diff
@@ -22,7 +22,7 @@ namespace nntile::kernel::embedding
 // Fill embedding from vocabulary
 template<typename T>
 void cpu(Index m, Index n, Index k, Index k_start, Index k_size,
-        const Index *index, const T *vocab, T *embed)
+        const int64_t *index, const T *vocab, T *embed)
```

Comment:
why has ```Index``` become a bad type?

Reply:
It is not "bad", it is just implicitly convertible to any other type. NNTile wrapped type `nntile::int64_t` does not allow implicit conversions and allows only explicit conversion to a basic `std::int64_t` type. This is to guarantee that we never do any arithmetic operations with our types. Just a data storage, nothing more. And a templated dispatching.